### DISCUSSION TITLE
How can I put an IP in the white list forever

### DISCUSSION THREAD

**Discussion by vahidalvandi** [docker-mailserver/docker-mailserver#3459] [Q&A] [answered]
How can I put an IP in the white list forever so that it is not blocked in dovecot and fail2ban?

**Comment by casperklein** [comment, marked as answer]
Use a custom [fail2ban-jail.cf](https://docker-mailserver.github.io/docker-mailserver/latest/config/advanced/optional-config/).

If you want to whitelist 1.2.3.4 for example, you can configure this:

```
[DEFAULT]

ignoreip = 127.0.0.1/8 1.2.3.4/32
```

**Comment by OE2WNL** [comment]
dovecot does not block ip's as far as I know - if you use RSPAMD within DMS we just discussed today about whitelist ip:
https://github.com/orgs/docker-mailserver/discussions/3613
